lib/beaker/host/unix.rb

```diff
@@ -58,12 +58,14 @@ def self.foss_defaults
 
     def self.aio_defaults
       h = self.foss_defaults
-      h['puppetbindir'] = '/opt/puppetlabs/agent/bin'
-      h['puppetpath'] = '/etc/puppetlabs/agent'
-      h['puppetvardir'] = '/opt/puppetlabs/agent/cache'
-      h['distmoduledir'] = '/opt/puppetlabs/agent/modules'
-      h['sitemoduledir'] = '/etc/puppetlabs/agent/modules'
-      h['hieraconf'] = ' /etc/puppetlabs/agent/hiera.conf'
+      h['puppetbindir']   = '/opt/puppetlabs/agent/bin'
+      h['puppetbin']      = "#{h['puppetbindir']}/puppet"
+      h['puppetpath']     = '/etc/puppetlabs/agent'
+      h['puppetvardir']   = '/opt/puppetlabs/agent/cache'
+      h['distmoduledir']  = '/opt/puppetlabs/agent/modules'
+      h['sitemoduledir']  = "#{h['puppetpath']}/code/modules"
```

Comment:
I bet we'll kick ourselves in the near future if we don't put in the new $codedir.

What do you think about adding `h['puppetcodedir'] = "#{h['puppetpath]}/code"` and then rooting these at `puppetcodedir`?